legend-engine-xts-sql/legend-engine-xt-sql-postgres-server/src/main/java/org/finos/legend/engine/postgres/config/HandlerType.java

```diff
@@ -17,5 +17,6 @@
 public enum HandlerType
 {
     LEGEND,
-    JDBC
+    JDBC,
+    STATIC
```

Comment:
Can we move all static handler code into test sources?

Reply:
this might  be hard, don't want to lose the config mapping. It is nice that we can map the config to the class.


@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
@JsonSubTypes({
        @JsonSubTypes.Type(value = LegendHandlerConfig.class, name = "LEGEND"),
        @JsonSubTypes.Type(value = JDBCHandlerConfig.class, name = "JDBC")
        @JsonSubTypes.Type(value = JDBCHandlerConfig.class, name = "JDBC"),
        @JsonSubTypes.Type(value = StaticHandlerConfig.class, name = "STATIC")
})

Reply:
Can we use [JsonTypeInfo.Id.CLASS](https://fasterxml.github.io/jackson-annotations/javadoc/2.6/com/fasterxml/jackson/annotation/JsonTypeInfo.Id.html#CLASS) instead?